File: Python/Ex27.py
n = float (input('Digite um valor positivo: '))

while (n <= 0):
    print (" Erro! Apenas números positivos. ")
    n = float (input(' Digite novamente:')) 

A = float (input('Digite o primeiro valor: '))
B = float (input('Digite o segundo valor: '))


while (B < A):
    print (" Erro! O segundo valor deve ser maior que o primeiro. ")
    B = int (input(' Digite novamente:')) 

i= B        

while (i >= A):
    T= n*i
    
    print(f'{n} X {i} = {T}')
    i= i-1

File: C/EX27.c
#include <stdio.h>

int main(int argc, char *argv[])
{
  int x, a, b, i, t;
 
  printf("Digite um numero: ");
  scanf("%i", &x);
 
  while(x <= 0)
  {
    printf("Erro! Apenas numeros positivos. Digite novamente: ");
    scanf("%i", &x);
  }
  printf("Digite o primeiro valor do intervalo: ");
  scanf("%i", &a);
  
  printf("Digite o segundo valor do intervalo: ");
  scanf("%i", &b);
 
  while(b < a)
  {
    printf("Erro! O segundo valor tem que ser maior que o primeiro. Digite novamente: ");
    scanf("%i", &b);
  }

  i=b;
    while(i >= a){
    t = x * i;
    printf("%i x %i = %i\n", x, i, t);
    i = i - 1;
  }
 
  return 0;
}
